pkg/loadbalance/weighted_round_robin.go

```diff
@@ -46,16 +46,18 @@ func newWeightedRoundRobinPicker(instances []discovery.Instance) Picker {
 	wrrp.nodes = make([]*wrrNode, wrrp.size)
 	offset := fastrand.Uint64n(wrrp.size)
 	totalWeight := 0
+	gcd := 0
```

Comment:
Please add some test cases, for example, if `instances` is an empty list, your code may crash.

Reply:
All right, it seems no use case that `newWeightedRoundRobinPicker` with empty list

https://github.com/cloudwego/kitex/blob/9618cfa5522e73f21bbee35a91af5aea6127605f/pkg/loadbalance/weighted_balancer.go#L74-L112